docs/reference/ml/trained-models/apis/start-trained-model-deployment.asciidoc

```diff
@@ -23,8 +23,7 @@ Requires the `manage_ml` cluster privilege. This privilege is included in the
 [[start-trained-model-deployment-desc]]
 == {api-description-title}
 
-Currently only `pytorch` models are supported for deployment. When deployed,
-the model attempts allocation to every machine learning node.
+When you deploy the model, it is allocated to all available {ml} nodes.
```

Comment:
We need to call out that only pytorch models are supported. boosted tree models, or models built by data frame analytics cannot be "deployed"

Reply:
Thanks for the clarification!  I've reworded a bit, let me know if that's not accurate.